plugins/activity/plugin.go

```diff
@@ -39,8 +39,13 @@ func configure(plugin *node.Plugin) {
 // broadcastActivityBlock broadcasts a sync beacon via communication layer.
 func broadcastActivityBlock() {
 	activityPayload := payload.NewGenericDataPayload([]byte("activity"))
-
-	time.Sleep(deps.BlockIssuer.Estimate())
+	for {
+		if estimate := deps.BlockIssuer.Estimate(); estimate > 0 {
+			time.Sleep(estimate)
+		} else {
+			break
+		}
+	}
```

Comment:
Could we end up in a something like infinite loop here? Should we introduce some value that limits the total time waiting? Or number of times we wait? But on the other hand if it's told to wait all the time, that means that the node is issuing other blocks, so activity blocks are not really necessary. So I think it's safe to leave it as is. WDYT?

Reply:
I think this could result in very long sleep times in the case of low mana nodes, but activity blocks should not really be goverened by the congestion control for regular blocks in the long run anyway, they should be part of a separate allocation for validators. I think it is fine to leave it as it is for now.